pkgs/applications/misc/pyditz/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, python27, pythonPackages }:
+
+python27.pkgs.buildPythonApplication rec {
```

Comment:
You can get the python package from ```pythonPackages.python``` instead of ```python27```. That way you know python and the package set will be in sync.

And ```python27.pkgs.buildPythonApplication``` can be changed to ```pythonPackages.buildPythonApplication```.